src/quo2/components/counter/step/__tests__/step_component_spec.cljs

```diff
@@ -0,0 +1,19 @@
+(ns quo2.components.counter.step.--tests--.step-component-spec
+  (:require [quo2.components.counter.step.view :as step]
+            [test-helpers.component :as h]))
+
+(h/describe "step component"
+  (h/test "default render of step component"
+    (h/render [step/step {} nil])
+    (-> (h/expect (h/get-by-test-id :step-component))
```

Comment:
prefer to use accessibility over a test ID, this is a best practice from the rtl team